src/routes/user/user.service.ts

```diff
@@ -6,13 +6,15 @@ import { generateTokens } from 'src/utils/generateToken';
 import Format from 'src/utils/format';
 import { sendEmail } from 'src/services/mailing';
 import env from 'src/configs/envVars';
+import { UserDoc } from '../../../models/User';
```

Comment:
can we make it absolute import to make sure when anyone copy this code and paste anywhere, no need to change the paths?

Reply:
Okay i will update this